gui/qt/main_window.py

```diff
@@ -3766,7 +3765,7 @@ def on_slptok_pref(x):
             self.update_cashaddr_icon()
             self.update_tabs()
 
-        enable_slp = bool(self.config.get('enable_slp'))
+        enable_slp = bool(self.config.get('enable_slp', True))
```

Comment:
Here you coerced `enable_slp` excplicitly to bool -- probably so you can support the `==True` `==False` thing in other code ? :) Or maybe not.. 

So basically: it's a good idea not to compare to exactly `True` or exactly `False` (even though it works most of the time so long as the thing you're comparing is `bool`)

Consider:

```
myflag = 0

if not myflag:
    print("Well.. definitely is not True")

if myflag == False:
   print("Sooo.. must be False, right?)
```

It's generally easier to read the first clause and the second won't even evaluate right (it will not take that branch)... but whatever works.  One can certainly build great programs with whatever style, or horrible programs with good style.  Just my two cents about what's common/considered "cool" and what isn't. 👍